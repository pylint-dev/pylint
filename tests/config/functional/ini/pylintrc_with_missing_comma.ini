# Check that we raise an warning when commas are missing in INI .pylintrc file
# See https://github.com/PyCQA/pylint/issues/3122
[messages control]
disable = logging-not-lazylogging-format-interpolation
enable = locally-disabled
         suppressed-message
jobs = 10
reports = yes
