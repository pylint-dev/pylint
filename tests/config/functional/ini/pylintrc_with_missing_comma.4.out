************* Module {abspath}
{relpath}:1:0: W0012: Unknown option value for '--disable', expected a valid pylint message and got 'logging-not-lazylogging-format-interpolation' (unknown-option-value)
{relpath}:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'locally-disabled
suppressed-message' (unknown-option-value)
